packages/devtools/lib/src/performance/performance_screen.dart

```diff
@@ -0,0 +1,234 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:meta/meta.dart';
+
+import '../framework/framework.dart';
+import '../profiler/cpu_profile_flame_chart.dart';
+import '../profiler/cpu_profile_model.dart';
+import '../profiler/cpu_profile_tables.dart';
+import '../profiler/cpu_profiler.dart';
+import '../ui/custom.dart';
+import '../ui/elements.dart';
+import '../ui/html_icon_renderer.dart';
+import '../ui/icons.dart';
+import '../ui/material_icons.dart';
+import '../ui/primer.dart';
+import '../ui/theme.dart';
+import 'performance_controller.dart';
+
+const Icon record = MaterialIcon(
```

Comment:
should we move these icons to the shared ui directory?

Reply:
moved them to the MaterialIcons file (where Material icons like Clear and Exit live)